_posts/2019-12-11-google-cloud-loader-0.2.0-released/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Google Cloud Loader 0.2.0 released"
+title-short: Google Cloud Loader 0.2.0
+tags: [snowplow, gcs, gcp, loader ]
+author: Ben
+category: Releases
+permalink: /blog/2019/12/11/gcs-loader-0.2.0-released/
+---
+
+We are pleased to announce [version 0.2.0][release-020] of Google Cloud Storage loader.
+This small release is centered around critical bug fixes and updated default configuration.
```

Comment:
This is missing a "that allows you to....". If I'm recalling correctly, this is the release that brings the GCS loader performance up to standard to sink events from the collector, right? Would be good to pull that out as a benefit, in addition to any performance stats we have still.

Reply:
I added more info on performance. The bottom line is that performance should no longer be outrageously low but the application itself has a strong (performance) [competitor](https://cloud.google.com/blog/products/data-analytics/simple-backup-and-replay-streaming-of-events-using-cloud-pubsub-cloud-storage-and-cloud-dataflow) that we won't be able to beat when it comes to raw numbers. So I want the impression to be that the performance is way better (not ~10% of collector throughput) and definitely worth pointing out but it's not something we can brag about.

Reply:
Makes sense. I think what you've added speaking to the reason why we've made the change is more than enough. Thanks.